roles/tendrl-alerting/tasks/main.yml

```diff
@@ -2,11 +2,14 @@
 # installation
 #
 
-- include: source.yml
-  when: install_from == "source"
+# TODO Add when packages are ready
+#- name: Install tendrl-alerting
+#  yum:
+#    name=tendrl-alerting
+#    state=latest
 
-- include: binary.yml
-  when: install_from == "packages"
+- fail: msg="There are no packages yet, we can't start and enable it"
+  when: not ignore_incomplete_installation
```

Comment:
I would leave this role here. I will just update it and when they describe it better, we can use it and test it. According to [this](https://github.com/Tendrl/alerting/blob/develop/doc/source/installation.rst) it looks like it sends mails with reports and errors. But there is no mention in official [installation guide](https://github.com/Tendrl/documentation/wiki/Tendrl-Package-Installation-Reference).